lib/config/presets/internal/workarounds.ts

```diff
@@ -260,6 +268,17 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  replacementsMapping: {
```

Comment:
I'd expect this name to at least contain "mvdan-sh".

Reply:
So the rule name vs rules is 1-1? I was expecting to add more rules into the list.

Reply:
The "breaking" and "drop-in" presets are on the same level. In my understanding they would both have an `extends` array with the full set of presets that are either breaking or drop in, all named to generally indicate what they're doing.

Reply:
I'd suggest "mvdan-sh-to-sh-syntax"